### PR TITLE
Use custom handler to allow bypass ssl in Fortify SSC

### PR DESCRIPTION
**Pull request by Gee19**
Refs #SUP-52

### CODE REVIEW

File: sdetools/modules/import_fortify/fortify_integrator.py [hunk 1]
```diff
@@ -29,6 +29,7 @@ def __init__(self, config):
         config.opts.add('analysis_user', 'Fortify SSC user', default='')
         config.opts.add('analysis_pass', 'Fortify SSC password', default='')
         config.opts.add('analysis_api_token', 'Fortify SSC authtoken (AnalysisDownloadToken permission)', default='')
+        config.opts.add('analysis_validate_cert', 'Fortify SSC bypass server certificate validation')
```
Comment (RouganStriker) [inline sdetools/modules/import_fortify/fortify_integrator.py]: I think we should default this to true

File: sdetools/modules/import_fortify/fortify_ssc_importer.py [hunk 2]
```diff
@@ -31,6 +32,15 @@ def marshalled(self, context):
         header.set('axis2ns1:token', self.auth_token)
 
 
+class IgnoreSSLTransport(HttpAuthenticated):
```
Comment (RouganStriker) [inline sdetools/modules/import_fortify/fortify_ssc_importer.py]: Can we add a small commenting explaining how this works?

**Comment by RouganStriker** [comment]
I saw it work so +1 after the small changes

**Comment by geoffwhittington** [comment]
thanks @RouganStriker for reviewing

Good job @Gee19 !

**Pull request merged**